boards/beagle/beagleconnect_freedom/board_antenna.c

```diff
@@ -84,28 +83,23 @@ static void board_cc13xx_rf_callback(RF_Handle client, RF_GlobalEvent events, vo
 
 	if (events & RF_GlobalEventRadioSetup) {
 		/* Decode the current PA configuration. */
-		RF_TxPowerTable_PAType paType = (RF_TxPowerTable_PAType)
-			RF_getTxPower(client).paType;
+		RF_TxPowerTable_PAType paType =
+			(RF_TxPowerTable_PAType)RF_getTxPower(client).paType;
 		/* Decode the generic argument as a setup command. */
 		RF_RadioSetup *setupCommand = (RF_RadioSetup *)arg;
 
 		switch (setupCommand->common.commandNo) {
-		case (CMD_RADIO_SETUP):
-		case (CMD_BLE5_RADIO_SETUP):
+		case CMD_RADIO_SETUP:
+		case CMD_BLE5_RADIO_SETUP:
 			loDivider = RF_LODIVIDER_MASK & setupCommand->common.loDivider;
-			/* Sub-1GHz front-end. */
-			if (loDivider != 0)
-				sub1GHz = true;
 			break;
-		case (CMD_PROP_RADIO_DIV_SETUP):
+		case CMD_PROP_RADIO_DIV_SETUP:
 			loDivider = RF_LODIVIDER_MASK & setupCommand->prop_div.loDivider;
-			/* Sub-1GHz front-end. */
-			if (loDivider != 0)
-				sub1GHz = true;
 			break;
 		default:
 			break;
 		}
+		sub1GHz = (loDivider != 0);
```

Comment:
Well, this is more of a formatting change than a functional one. If you look at the code, it has the exact same effect as before. And well, the reason for doing this is to reduce diff with launchxl.

Reply:
This is definitely not a formatting change. It may be a refactoring that leaves functionality intact but that's true for all or most of your changes. This clearly belongs to the previous commit where your other refactorings have been placed, that also reduce the diff with LaunchXL. I've marked this as an optional change but that doesn't mean that we should dilute arguments.

Reply:
Moved